testkit/tests/counter/main.rs

```diff
@@ -627,11 +643,22 @@ fn test_explorer_transaction_statuses() {
         TxIncrement::new(&pubkey, u64::max_value() - 3, &key)
     };
 
-    testkit.create_block_with_transactions(txvec![
+    let block = testkit.create_block_with_transactions(txvec![
         tx.clone(),
         error_tx.clone(),
         panicking_tx.clone(),
     ]);
+    assert!(block[0].status().is_ok());
+    assert!({
+        let err = block[1].status().unwrap_err();
+        err.error_type() == ErrorType::Code(0) &&
+            err.description() == Some("Adding zero does nothing!")
+    });
+    assert!({
+        let err = block[2].status().unwrap_err();
+        err.error_type() == ErrorType::Panic &&
```

Comment:
It could make sense to write some helpers, e.g., `err.is_panic()` and `err.has_code(..)`.

Reply:
Perhaps, `assert_matches` can help?